stack.h

```diff
@@ -42,13 +42,15 @@ class runnable;
 
 extern bool indebugger;
 
+typedef std::pair<string, string> importIndex_t;
```

Comment:
from Andy: "[T]he pair of strings is sometimes called an index as importIndex_t and in other places only the second string is called an index."

Reply:
We've set things up to use sigHandle to specifically refer to the handle on the signature. We use Index to refer to an ordered pair (filename, sigHandle). The code has been revised to follow this convention consistently.